src/hotspot/cpu/riscv/interp_masm_riscv_64.cpp

```diff
@@ -80,22 +82,37 @@ void InterpreterMacroAssembler::dispatch_via(TosState state, address* table) {
 // Dispatch code executed in the prolog of a bytecode which does not do it's
 // own dispatch. The dispatch address is computed and placed in R24_dispatch_addr_PPC.
 void InterpreterMacroAssembler::dispatch_prolog(TosState state, int bcp_incr) {
-  Register bytecode = R6_scratch2;
```

Comment:
As far as I understand, we are not sure if we can allocate a temp register, so we do everything in `epilog`.
Then, there is no need to do anything in `prolog`. It's perfectly fine, for example, x86 doesn't do anything here.
I suggest to remove this prolog code completely.